src/code4ved/exceptions.py

```
@@ -1,50 +1,50 @@
-"""Custom exceptions for LCM Automation."""
+"""Custom exceptions for Code4Ved Automation."""
 
 
-class LCMError(Exception):
-    """Base exception for LCM Automation."""
+class Code4VedError(Exception):
+    """Base exception for Code4Ved Automation."""
 
     def __init__(self, message: str, code: str = None):
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)